src/main/java/core/basesyntax/PasswordValidator.java

```diff
@@ -1,7 +1,17 @@
 package core.basesyntax;
 
+import core.basesyntax.exception.PasswordValidationException;
+
 public class PasswordValidator {
-    public boolean validate(String password, String repeatPassword) {
-        return false;
+    public boolean validate(String password, String repeatPassword)
+            throws PasswordValidationException {
+        if (password != null
+                && password.equals(repeatPassword)
+                && password.length() >= 10
+                && repeatPassword.length() >= 10) {
```

Comment:
we don't need to check the `repeatPassword` for length - we already have `password.equals(repeatPassword)` part that handles this